highfive/configs/rust-lang/rust.json

```diff
@@ -38,15 +38,26 @@
     },
     "mentions": {
         "src/librustdoc/html/static": {
-            "message": "Some changes occurred in HTML/CSS.",
+            "message": "Some changes occurred in HTML/CSS/JS.",
+            "command": "@highfive: run-doc-ui",
+            "reviewers": ["@GuillaumeGomez"]
+        },
+        "src/librustdoc/html/static/themes": {
+            "message": "Some changes occurred in HTML/CSS themes.",
+            "command": "@highfive: run-doc-ui",
             "reviewers": ["@GuillaumeGomez"]
         },
         "src/src/doc": {
```

Comment:
Unrelated, but I'm pretty confident this path is wrong. Also, do you want to run the test suite every time the docs are updated?

Reply:
I just thought about it afterwards: it's pretty useless to run it on this path. Also, the correct path is indeed `src/doc`. I'll fix both.